tests/mpesa-rust/transaction_reversal_test.rs

```diff
@@ -20,16 +22,23 @@ async fn transaction_reversal_success() {
         .mount(&server)
         .await;
     let response = client
-        .transaction_reversal("testapi496")
-        .result_url("https://testdomain.com/ok")
-        .timeout_url("https://testdomain.com/err")
+        .transaction_reversal()
+        .initiator("testapi496")
+        .try_result_url("https://testdomain.com/ok")
+        .unwrap()
+        .try_timeout_url("https://testdomain.com/err")
+        .unwrap()
         .transaction_id("OEI2AK4Q16")
         .amount(1.0)
         .receiver_party("600111")
+        .receiver_identifier_type(IdentifierTypes::ShortCode)
```

Comment:
Made an interesting discovery about this field in #90

Reply:
Copied your values